empress_gui.py

```diff
@@ -4,362 +4,819 @@
+        App.recon_graph = None 
+        App.clusters_list = []
+        App.medians = None
+        # Reset self.recon_input so self.view_cost_space_btn can be disabled
+        self.recon_input = input_reader.ReconInput()
+        self.view_cost_space_btn.configure(state=tk.DISABLED)
+        self.view_tanglegram_btn.configure(state=tk.DISABLED)
+
+        if event == "Load host tree file":
+            self.load_files_dropdown['menu'].entryconfigure("Load parasite tree file", state = "normal")
+
+            # Read in host tree file after reseting everything
+            self.recon_input.read_host(self.host_file_path)
+
+            self.host_tree_info.destroy() 
+            self.parasite_tree_info.destroy()
+            self.mapping_info.destroy()
+
+        elif event == "Load parasite tree file":
+            self.load_files_dropdown['menu'].entryconfigure("Load mapping file", state = "normal")
+
+            # Read in host and parasite tree files after reseting everything
+            self.recon_input.read_host(self.host_file_path)
+            self.recon_input.read_parasite(self.parasite_file_path)
+
+            self.parasite_tree_info.destroy()
+            self.mapping_info.destroy()
+
+        elif event == "Load mapping file":
+            # Read in host and parasite trees and mapping files after reseting everything
+            self.recon_input.read_host(self.host_file_path)
+            self.recon_input.read_parasite(self.parasite_file_path) 
+            self.recon_input.read_mapping(self.mapping_file_path)
+
+            self.mapping_info.destroy()
+
+        # Reset dtl costs so self.compute_reconciliations_btn can be disabled
         self.dup_cost = None
         self.trans_cost = None
         self.loss_cost = None
+        self.dup_input.set(1.00)
+        self.trans_input.set(1.00)
+        self.loss_input.set(1.00)
+        self.compute_reconciliations_btn.configure(state=tk.DISABLED)
+        self.view_solution_space_dropdown.configure(state=tk.DISABLED)
+        self.view_reconciliations_dropdown.configure(state=tk.DISABLED)
+        self.view_pvalue_histogram_btn.configure(state=tk.DISABLED)
+
+        # Reset the rest
+        self.dup_label.destroy()
+        self.dup_entry_box.destroy()
+        self.trans_label.destroy()
+        self.trans_entry_box.destroy()
+        self.loss_label.destroy()
+        self.loss_entry_box.destroy()
+
+        self.recon_MPRs_label.destroy()
+        self.num_MPRs_label.destroy()
+        self.recon_cospeci_label.destroy()
+        self.recon_dup_label.destroy()
+        self.recon_trans_label.destroy()
+        self.recon_loss_label.destroy()
+
+        self.num_cluster_input = tk.IntVar()
+        self.num_cluster_input.set(1)
+        self.num_cluster = None
+
+        self.recon_info_displayed = False
+        
+        if self.cost_space_window is not None and self.cost_space_window.winfo_exists() == 1:
```

Comment:
Remove the `== 1`. `1` evaluates to True and `0` to False

Reply:
done